### PR TITLE
Enhancement/fix linting errors

### PR DESCRIPTION
**Pull request by swashcap**
# Problem

Linting fails! :scream:

referenced issue(s): _n/a_

# Solution

Fix the linting errors one-by-one.

# Testing

```shell
npm run lint
```

### CODE REVIEW

File: packages/coinstac-images/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (swashcap) [inline packages/coinstac-images/.eslintrc.js:1]: Assuming all these subpackages are built as Docker images, so they won't have a _node_modules_ dir, causing the eslint-plugin-import to error. Disabling the rule seems like a sensible action.

File: packages/coinstac-ui/app/render/state/graphql/props.js [hunk 2]
```diff
@@ -27,8 +27,9 @@ export const consortiaMembershipProp = (name) => {
             const consQuery = store.readQuery({ query: FETCH_ALL_CONSORTIA_QUERY });
             const index = consQuery.fetchAllConsortia.findIndex((cons) => {
               if (cons && data[name]) {
-                cons.id === data[name].id;
+                return cons.id === data[name].id;
```
Comment (swashcap) [inline packages/coinstac-ui/app/render/state/graphql/props.js:30]: This changes program behavior! Could this be a bug where `index` is always `-1`?

File: packages/coinstac-images/.eslintrc.js (hunk 1 above)
Comment (rssk) [reply packages/coinstac-images/.eslintrc.js:1]: Yeah... I've been on the fence of how they should work, I think they're just fine as a free floating dir in packages that are then build as images. Maybe some update scripts to keep things consistent and make pubing images easier

File: packages/coinstac-ui/app/render/state/graphql/props.js (hunk 2 above)
Comment (rssk) [reply packages/coinstac-ui/app/render/state/graphql/props.js:30]: Great catch! I wonder... there are quite a few odd UI bugs

**Review by rssk — APPROVED** [review]

File: packages/coinstac-ui/app/render/state/graphql/props.js (hunk 2 above)
Comment (swashcap) [reply packages/coinstac-ui/app/render/state/graphql/props.js:30]: Thank the magic of ESLint 🔮

**Pull request merged**